spec/lib/data_update_scripts/update_default_email_addresses_spec.rb

```diff
@@ -0,0 +1,36 @@
+require "rails_helper"
+require Rails.root.join(
+  "lib/data_update_scripts/20201218080343_update_default_email_addresses.rb",
+)
+
+describe DataUpdateScripts::UpdateDefaultEmailAddresses do
+  let(:contact_email) { "contact@dev.to" }
+
+  before do
+    allow(SiteConfig).to receive(:email_addresses).and_return(
+      {
+        default: "hi@dev.to",
+        business: "business@dev.to",
+        privacy: "privacy@dev.to",
+        members: "members@dev.to"
+      },
+    )
+  end
+
+  context "with no contact email set in the email_addresses hash" do
+    it "adds the contact email" do
+      described_class.new.run
+      puts SiteConfig.email_addresses
```

Comment:
I'll remove this log, but I've been trying to get this test to pass without success. Can I get another set of eyes on it please, maybe @rhymes if you have some capacity please? 

I tested the changes manually as per my QA instructions and I feel certain that the data script will add the contact email to the database, however, I feel like I'm missing something (maybe obvious?) in the tests.

Reply:
✅